src/handlers/comment/handlers/unassign.ts

```diff
@@ -2,7 +2,7 @@ import { removeAssignees } from "../../../helpers";
 import { getBotContext, getLogger } from "../../../bindings";
 import { Payload } from "../../../types";
 import { IssueCommentCommands } from "../commands";
-import { closePullRequestForAnIssue } from "../..";
+import { closePullRequestForAnIssue } from "../../assign/index";
```

Comment:
the changes is different from the original one???

Reply:
lol not sure about this
Anyway just wanna have a look if it is fixed on some hunter's local with this pr.

Reply:
yeah it's working

Reply:
Cool. @0xcodercrane Seems we can merge this.